src/containers/App/assets/app.scss

```diff
@@ -1,8 +1,9 @@
-@import '~theme/base/variables';
+@import '../../../theme/base/variables';
```

Comment:
I'm assuming this is because storybook? Would it be possible to make storybook to resolve the `src` folder? 🤔

Reply:
It's also a matter of resolving the `~`. I will need to look into it some more